card/src/main/java/com/adyen/checkout/card/DefaultCardDelegate.kt

```diff
@@ -84,6 +87,15 @@ internal class DefaultCardDelegate(
     private val _outputDataFlow = MutableStateFlow(createOutputData())
     override val outputDataFlow: Flow<CardOutputData> = _outputDataFlow
 
+    override val addressOutputData: AddressOutputData
+        get() = outputData.addressState
+
+    override val addressOutputDataFlow: Flow<AddressOutputData> by lazy {
```

Comment:
Why do we need `by lazy` here?

Reply:
If this variable is not lazy, it runs before we set the coroutinescope in the initialize method, so it crashes beacuse coroutineScope is null. I need to run this line after the initialize method, so I set it lazy to prevent a crash.